code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -59,30 +58,31 @@
 		countdown = null
 	. = ..()
 
-/obj/structure/destructible/clockwork/massive/celestial_gateway/destroyed()
-	countdown.stop()
-	visible_message("<span class='userdanger'>The [src] begins to pulse uncontrollably... you might want to run!</span>")
-	world << sound('sound/effects/clockcult_gateway_disrupted.ogg', 0, channel = 8, volume = 50)
-	make_glow()
-	glow.icon_state = "clockwork_gateway_disrupted"
-	takes_damage = FALSE
-	sleep(27)
-	explosion(src, 1, 3, 8, 8)
+/obj/structure/destructible/clockwork/massive/celestial_gateway/deconstruct(disassembled = TRUE)
```

Comment:
I'm changing all /obj/ that have some form of deconstruction proc to all use the same proc: deconstruct(), with an arg to separate being disassembled nicely from being smashed to pieces.